test/runnable/cdcmp.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+src_file=${EXTRA_FILES}/${TEST_NAME}.d
+expect_file=${EXTRA_FILES}/${TEST_NAME}.out
+tmp_file=${RESULTS_TEST_DIR}/${TEST_NAME}.out
+obj_file=${RESULTS_TEST_DIR}/${TEST_NAME}.o
+
+if [ $OS == "linux" ] && [ $MODEL == "64" ]; then
+  $DMD -c -O ${src_file} -of${obj_file}
+  objdump --disassemble --disassembler-options=intel-mnemonic ${obj_file} | tail -n+3 > ${tmp_file}
+  diff ${expect_file} ${tmp_file}
+  rm ${obj_file} ${tmp_file}
```

Comment:
```suggestion
  rm -rf "${RESULTS_TEST_DIR}"
```

Reply:
That change appears to cause other problems with missing files.

Reply:
I just realized that your suggested change does not work because multiple tests are run in parallel, meaning blowing away the results directory randomly deletes the work of other tests in progress.

Reply:
Nope, not at all. Just use `OUTPUT_BASE` l though you were using. Sorry for not noticing that. `OUTPUT_BASE` is a directory which is unique for each test.

See also: https://github.com/dlang/dmd/blob/master/test/tools/exported_vars.sh